tools/cosmovisor/README.md

```diff
@@ -32,7 +32,7 @@ Cosmovisor is designed to be used as a wrapper for a `Cosmos SDK` app:
 
 *Note: If new versions of the application are not set up to run in-place store migrations, migrations will need to be run manually before restarting `cosmovisor` with the new binary. For this reason, we recommend applications adopt in-place store migrations.*
 
-*Note: If validators would like to enable the auto-download option (which [we don't recommend](#auto-download)), and they are currently running an application using Cosmos SDK `v0.42`, they will need to use Cosmovisor [`v0.1`](https://github.com/cosmos/cosmos-sdk/releases/tag/cosmovisor%2Fv0.1.0). Later versions of Cosmovisor do not support Cosmos SDK `v0.44.3` or earlier if the auto-download option is enabled.*
+*Note: Only the last two versions (v1.3.0 & v1.5.0) of cosmovisor are actively developed/maintained. Older versions are longer maintained and is recommended not to be used.*
```

Comment:
We actually maintain only one version due to how cosmovisor is released. So imho we should say to always use the latest version.